packages/fxa-content-server/app/tests/spec/views/survey.js

```diff
@@ -0,0 +1,57 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ import _ from 'underscore';
+ import $ from 'jquery';
+ import { assert } from 'chai';
+ import View from 'views/survey';
+
+ describe('views/survey', function() {
+   var view;
+
+   function createView(options) {
+     var viewOptions = _.extend(
+       {
+         surveyURL: 'https://www.surveygizmo.com/s3/5541940/pizza',
+         viewName: 'survey',
+       },
+       options || {}
+     );
+     return new View(viewOptions);
+   }
+
+   beforeEach(function() {
+     view = createView();
+     return view.render().then(function() {
+       $('#container').html(view.el);
+     });
+   });
+
+   afterEach(function() {
+     view.remove();
+     view.destroy();
+     $('#container').empty();
+   });
+
+
+   describe('render', function() {
+     it('renders template', function() {
+       assert.ok($('.survey-wrapper').length);
+     });
+
+     it('shows the iframe', function() {
+       view = createView();
+
+       return view.render().then(() => {
+         assert.lengthOf(view.$('iframe'), 1);
+       });
+     });
+   });
+
+   describe('afterRender', () => {
+     beforeEach(() => {
+       return view.afterRender();
+     });
+   });
+ });
```

Comment:
In addition to `SurveyWrapperView`, there's also a function component `SurveyView` in `survey.js`.  I don't see any tests for `SurveyView`.

Reply:
Hmm, I'm having trouble thinking of a meaningful test for `SurveyView`. `CreateHandleIframeTask` is covered in `fxa-react` and SurveyView view just renders `Survey` which is tested on line 46.

Reply:
Sorry, I missed this and it's not displayed on diffs on newer commits.  Anyways, iirc there's an event handler being registered.  If we spy on that, we could see if the registration happened, what the handler is (if possible...I'm writing this without looking at the code), and the effects when the handler is called.